### ISSUE TITLE
Laser Creepers

### ISSUE THREAD

**Issue by andr9528** [Darkosto/InvasionIssues#121]
through the configs they are set to only spawn in groups of 1, but a group of 9 just walked past my window O.O.

also they still seem to spawn regardless og light level - seen them spawn in perfectly light area from torches

**Comment by andr9528** [comment]
oh and they can also still shot for a short time after they have died...

**Comment by GenDeathrow** [comment]
Seeing that @vadis365 and I got together and fixed all spawning mobs, this shouldn't be the case. 

As far as the group size, just because the group size is one does not mean more than one creep can spawn in a chunk.. that means 9 creepers happen to spawn not a single group of 9. 

Vadis can give more insight on this.

**Comment by andr9528** [comment]
Ya i know that you already have worked on this, which i can fell, as they do not spawn as often as they did before, but i think thats more because of the fact that raiders are spawning correctly now.

i'm not on the PC that have the game files right now (+ the next 5-6 hours) but when i get home i will first make a comment with a picture showing where they spawned and where i where when they did, as i where looking Straight at the spot they spawned at, while dead.

Then i'm going to make a little tour of my base showing that it is all lit up, and inacceable to mobs that do not fly, and then i'm going to wait for laser creepers to spawn in said base, all of that while recording it, as prof of the issue.

**Comment by andr9528** [comment]
![2017-04-06_14 16 36](https://cloud.githubusercontent.com/assets/21788044/24753730/b6418be6-1ad3-11e7-9af7-1cd34b3a9245.png)

the picture shows where i saw a group of 3 creepers spawn in at once, marked with the redstone torch, my location is the spot where i were at when they spawned while being dead

the torches is set up in a perfect grid pattern and the picture was taken with F7 on

**Comment by vadis365** [comment]
Hello,
In response to this issue I have changed some more things in the code for the Laser Creepers but the spawning rules are essentially the same as every other mob and should honor the vanilla block lighting rules. Something that was happening, which I found out, was the T-R3X was not spawning correct. The problem may have been connected as I observed the dino spawn in and a creeper was then mounted but then the dino would instantly de-spawn and leave the creeper. This may have been what happened.
I have to fix another error in the code in which I have left some code in which shouldn't be there (involving Zombies spawning with laser blasters) but there should be a fix up later.
Cheers,
Me.
